doc/source/contributing.rst

```diff
@@ -3,35 +3,42 @@
 Contributing
```

Comment:
Is this file needed now that you included contributing.md? Seems like it's duplicate text which I feel the same about as with code. Not a huge issue, but they might not be identical in the future.

Reply:
I totally get your point, and I tend to agree. However, I am not really sure where I think this information mainly belongs, so for now I would like to keep it this way. I mainly put the `CONTRIBUTING.md` files because it fits in to the whole GitHub experience :). 

If we find that we want to change these files often then we should just pick a single location though.